docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -135,6 +135,8 @@ Property                                              Description
                                                       enabling this feature.
 ``bigquery.view-materialization-project``             The project where the materialized view is going to be created The view's project
 ``bigquery.view-materialization-dataset``             The dataset where the materialized view is going to be created The view's dataset
+``bigquery.views-cache-ttl``                          Duration for which the results of querying a view will be      ``15m``
```

Comment:
I think it'll be more correct to say something like:

`Duration for which the materialization of a view will be cached and reused.`

I think we only cache the name of the materialized table and if it's present in cache we use it. If not we materialize the view again?

Reply:
Right. Updated including the config description.